docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
+#### What happens to slashed tokens?
 
-* We will distribute a total of ~1,5 billion tokens as staking rewards in ~10 years.
+Slashed tokens are sent to the community pool.
 
-* The exact number of tokens is given by the formula 
+:::warning Slashing
+When a validator’s tokens are slashed by a certain percentage, so are its delegators.
+:::
 
-**f(x) = 1.8 x^2 - 53 x + 358** 
+#### How to unjail
 
-**where *x* is in years and *f(x)* is in millions of CUDOS.**
+To unjail, the validator must send a transaction from the jailed validator account via the `Cudos-noded CLI`. 
 
-* Hence the amount to be distributed in a given time interval is obtained by integrating the above function over that interval.
+```shell
+cudos-noded tx slashing unjail --from mykey [flags]
+```
 
-* Due to the assumption that tx fees will eventually be the main source of revenue, staking rewards start high and then decrease quadratically in time.
+### Unbonding period
 
-:::tip Important
+If a validator is jailed or leaves the bonded validators set, it enters an **unbonding period**. A validator that has failed to get itself back into the bonded validators set *within 21 days* is said to be **unbonded**.
 
-Important note: Validators need to use the --commission flag in the CLI in order to withdraw their commission.
+:::caution 
+Slashing can still occur during the unbonding period.
 :::
 
-## Jailing/Slashing
+### Token holder staking
 
-* Tokens delegated to a bonded validator is slashed if there is evidence of the validator misbehaving.
+Anyone holding CUDOS tokens can delegate or stake their tokens to one or more **validator nodes** of their choice. Validators and Stakers then share rewards **based on the amounts they have contributed to the pool**.
 
-* This means that when a validator’s tokens are slashed by a certain percentage, so are it delegators’.
+Validator Operators can set their own Commission to charge Stakers.**Commission rates** can change *once every day* however, the **Maximum Commission Rate** cannot be exceeded. Commission fees can be viewed under [Validator details in the Cudos Explorer](https://explorer.cudos.org/validators). 
 
-* If a validator fails to sign 90% of the blocks within a 19200-block interval (i.e. 17280 blocks), then it is jailed:
+![validator-details](@site/static/img/validator-details.png)
 
-    * It becomes not-bonded and begins unbonding
-    * It stops signing and proposing blocks
-    * It stops earning block rewards
-    * It’s stake gets slashed by 0.01%
+## Redelegation of tokens
 
-* The validator needs to unjail itself after at least 600 seconds of being jailed. There is an unjail transaction that can be sent via the CLI from within the Validator account/node. 
-
-```shell
-cudos-noded tx slashing unjail --from mykey [flags]
-```
+A **staker/delegator** can choose to **undelegate** their tokens at any time. 
```

Comment:
Maybe mvoe the "undelegate" bit to outside of the "redelegate" section?

Reply:
done